Added an alert for form validation

rshin8763 opened the pull request.

rshin8763 committed:
Added an alert for form validation

rshin8763 committed:
added queue manager to front page

Added queue manager button to index.hbs

tymc7 commented on views/queue-manager.hbs:
```diff
@@ -52,16 +52,22 @@
-      <input type="number" min="0" step="1" name="pid" id="pid"><br>
+    <div class="form-row">
+        <div class="form-group col-md-4">
+            Patient ID &nbsp
```
should there be a `;` after the `&nbsp`?

tymc7 commented on views/queue-manager.hbs:
```diff
@@ -52,16 +52,22 @@
+            <input type="number" min="0" step="1" name="pid" id="pid"><br>
+        </div>
+        <div class="form-group col-md-4">
+            New Priority &nbsp
```
Same here

tymc7 reviewed (commented):
Also, this is more for discussion, but why choose non-breaking spaces rather than a `form-inline` class?

rshin8763 committed:
small changes.

Added ;. we can change this later, but this merge is required for me to test my edit queue functions.

ethanysc reviewed (commented):
Merging so you can test.
